djstripe/models/base.py

```diff
@@ -188,10 +188,23 @@ def _stripe_object_to_record(cls, data, current_ids=None, pending_relations=None
         # Iterate over all the fields that we know are related to Stripe,
         # let each field work its own magic
         ignore_fields = ["date_purged", "subscriber"]  # XXX: Customer hack
-        for field in cls._meta.fields:
+        for field in cls._meta.get_fields():
             if field.name.startswith("djstripe_") or field.name in ignore_fields:
                 continue
-            if isinstance(field, models.ForeignKey):
+            if isinstance(
```

Comment:
The preferred way of checking the field type is via checking field attributes according to the docs:
https://docs.djangoproject.com/en/2.2/ref/models/fields/#field-attribute-reference

Check throughout.

Reply:
Also if this is a reverse field I need to check it's from dj-stripe's app.